### PR TITLE
Add PHP 8.2 with Laravel 9.4 to relenv

### PR DESCRIPTION
**Pull request by realFlowControl**
### Description

This will add PHP 8.2 and Laravel 9.4 to relenv

### CODE REVIEW

**Comment by morrisonlevi** [comment]
What changes did you make to the Laravel app before committing it? Or are these the out-of-the-box sources?

File: dockerfiles/testing-environment/Laravel94/app/app/Http/Controllers/ExampleController.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+namespace App\Http\Controllers;
+
+use Illuminate\Routing\Controller as BaseController;
+use Illuminate\Support\Facades\DB;
+
+class ExampleController extends BaseController
+{
+    public function example()
+    {
+        return "Hi from Laravel 5.7 app!\n";
```
Comment (morrisonlevi) [inline dockerfiles/testing-environment/Laravel94/app/app/Http/Controllers/ExampleController.php:12]: Laravel 5.7, huh? :)

Comment (realFlowControl) [reply dockerfiles/testing-environment/Laravel94/app/app/Http/Controllers/ExampleController.php]: Nice find, but that was basically the source, we just copied the old version and updated it

**Comment by realFlowControl** [comment]
> What changes did you make to the Laravel app before committing it? Or are these the out-of-the-box sources?

Those were out of the box sources, but I changed this to use `composer create-project` to install this. The configuration is copied over from the Laravel 5.7 setup.

File: dockerfiles/testing-environment/Laravel94/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+# Install PHP
+ADD docker-scripts/install-php-and-composer.sh /scripts/install-php-and-composer.sh
+RUN /scripts/install-php-and-composer.sh
+
+RUN touch /var/log/php-fpm.log && chmod 777 /var/log/php-fpm.log
+
+# Setup web app
+WORKDIR /var/www/html/
+RUN rm -rf /var/www/html/* \
+    && composer create-project laravel/laravel=9.4.0 /var/www/html --prefer-dist \
```
Comment (morrisonlevi) [inline dockerfiles/testing-environment/Laravel94/Dockerfile:19]: > > What changes did you make to the Laravel app before committing it? Or are these the out-of-the-box sources?
> 
> Those were out of the box sources, but I changed this to use `composer create-project` to install this. The configuration is copied over from the Laravel 5.7 setup.

Thanks!

File: dockerfiles/testing-environment/Laravel94/docker-scripts/local-run.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+set -e
+
+echo "Starting up local testing supervisord."
+echo "Tracer to be installed: ${TRACER_DOWNLOAD_URL}"
+
+if [ ! -z ${TRACER_DOWNLOAD_URL} ]; then
+    echo "Installing tracer at: ${TRACER_DOWNLOAD_URL}"
+    wget -O dd-trace-php.deb ${TRACER_DOWNLOAD_URL}
+    dpkg -i dd-trace-php.deb
+fi
```
Comment (morrisonlevi) [inline dockerfiles/testing-environment/Laravel94/docker-scripts/local-run.sh:12]: How does the profiler get installed? I think we need to use `datadog-setup.php` and check for a profiling env var?

Also, does that URL get overwritten by the relenv jobs?

Comment (realFlowControl) [reply dockerfiles/testing-environment/Laravel94/docker-scripts/local-run.sh:12]: That's the neat part 🤯 
Basically in the [bazel deployment](https://github.com/DataDog/datadog-reliability-env/blob/master/apps/php/laravel/deployment.yaml) we add another layer with the [`run.sh`](https://github.com/DataDog/datadog-reliability-env/blob/master/apps/php/laravel/run.sh) and overwrite the `cmd` of the container. It's not nice, but this is just copied over from the other profiling containers in relenv.

**Comment by morrisonlevi** [comment]
Thank you for doing this! I moved this code to the relenv.

**Comment by r1viollet** [comment]
@realFlowControl were these changes integrated in another PR ?

**Pull request closed without merging**